cg/cli/workflow/mip_dna/base.py

```diff
@@ -173,85 +162,80 @@ def run(
     dry_run: bool = False,
     email: str = None,
     mip_dry_run: bool = False,
-    priority: str = "Normal",
+    priority: str = None,
     skip_evaluation: bool = False,
     start_with: str = None,
 ):
     """Run the analysis for a case"""
     dna_api = context.obj["dna_api"]
-    tb_api = context.obj["tb"]
 
-    email = email or environ_email()
+    if case_id is None:
+        _suggest_cases_to_analyze(context)
+        return
+    case_obj = dna_api.db.family(case_id)
+    if not case_exists(case_obj, case_id):
+        LOG.error(f"Case {case_id} does not exist!")
+        raise click.Abort()
 
+    email = email or environ_email()
     kwargs = dict(
         config=context.obj["mip-rd-dna"]["mip_config"],
         case=case_id,
-        priority=priority,
+        priority=priority or dna_api.get_priority(case_obj),
         email=email,
         dryrun=mip_dry_run,
         start_with=start_with,
         skip_evaluation=skip_evaluation,
     )
-    if case_id is None:
-        _suggest_cases_to_analyze(context)
-        context.abort()
 
-    case_obj = context.obj["db"].family(case_id)
-    if not case_exists(case_obj, case_id):
-        context.abort()
-    if tb_api.is_analysis_ongoing(case_id=case_obj.internal_id):
+    if dna_api.tb.is_analysis_ongoing(case_id=case_obj.internal_id):
         LOG.warning("%s: analysis is ongoing - skipping", case_obj.internal_id)
         return
+
+    if dna_api.get_skip_evaluation_flag(case_obj=case_obj):
+        kwargs["skip_evaluation"] = True
+
     if dry_run:
-        dna_api.run(dry_run=dry_run, **kwargs)
-    else:
-        dna_api.run(**kwargs)
-        tb_api.mark_analyses_deleted(case_id=case_id)
-        tb_api.add_pending_analysis(case_id=case_id, email=email)
-        LOG.info("MIP rd-dna run started!")
+        dna_api.run_command(dry_run=dry_run, **kwargs)
+        return
+
+    dna_api.run_command(**kwargs)
+    dna_api.tb.mark_analyses_deleted(case_id=case_id)
+    dna_api.tb.add_pending_analysis(case_id=case_id, email=email)
+    case_obj.action = "running"
```

Comment:
This should be an imported constant and not hard coded. That way it can be consistent throughout the code base